libcperciva/datastruct/seqptrmap.c

```diff
@@ -74,7 +79,18 @@ seqptrmap_add(struct seqptrmap * M, void * ptr)
 void *
 seqptrmap_get(struct seqptrmap * M, int64_t i)
 {
-	size_t pos = i - M->offset;
+	size_t pos;
+	int64_t pos_i;
+
+	/* Sanity check, but we cannot fail, so don't use an assert. */
+	if (i < 0)
```

Comment:
"Sanity check" means "if this function is used correctly, this will always be true".  Do we state that negative values of `i` should never be passed?  No, all we do is state the behaviour which applies in that case -- namely that we'll return NULL (since no pointer is ever associated with a negative value).